apps/roam/src/components/canvas/Tldraw.tsx

```
@@ -956,6 +931,10 @@ const InsideEditorAndUiContext = ({
   return <CustomContextMenu extensionAPI={extensionAPI} allNodes={allNodes} />;
 };
 
+const CanvasDrawerPanelWrapper = ({}: {}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)